conf/deployment/baremetal/upi_1az_rhcos_multus_cluster_nvme_intel_3m_3w.yaml

```diff
@@ -12,6 +12,7 @@ ENV_DATA:
   disk_pattern: 'nvme-INTEL'
   is_multus_enabled: true
   multus_public_net_interface: 'enp1s0f0'
+  multus_cluster_net_interface: 'enp1s0f0'
```

Comment:
shouldn't we change it to enp1s0f1?

Reply:
yes.. if it is woking...I will test it again manually..